tools/importPlanGoals.js

```diff
@@ -83,29 +103,38 @@ export default async function importGoals(file) {
             column = goalColumn[2].toLowerCase();
             if (column === 'topics') {
               const allTopics = el[key].split('\n');
-              allTopics.forEach((topicPipeRoles) => {
+              for await (const topicPipeRoles of allTopics) {
                 const topic = topicPipeRoles.split('|')[0];
                 const roles = topicPipeRoles.split('|')[1];
                 const trimmedTopic = topic.trim();
                 if (trimmedTopic !== '') {
-                  const topicId = getItemId(cleanTopics, trimmedTopic);
+                  const [dbTopic] = await Topic.findOrCreate({ where: { name: trimmedTopic } });
+                  const topicId = dbTopic.id;
                   if (roles) {
                     const rolesArr = roles.split(',');
-                    rolesArr.forEach((role) => {
+                    for await (const role of rolesArr) {
                       const trimmedRole = role.trim();
-                      const roleId = getItemId(cleanRoles, trimmedRole);
+                      let roleId;
+                      if (trimmedRole === 'GS') { // Special case for 'GS' since it's non-unique
+                        const [dbRole] = await Role.findOrCreate({ where: { name: trimmedRole, fullName: 'Grantee Specialist' } });
```

Comment:
Are we assuming all `GS` roles from the CSV are "Grantee Specialists" and not "Grant Specialists"?

Reply:
Yes, that is the assumption, which was confirmed with Ryan.